spinnaker-jenkins-to-vmss/setup-scripts/create_service_principal.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+#echo "Usage:
+#  1 bash create_service_principal.sh
+#  2 bash create_service_principal.sh <Subscription ID>
+
+SUBSCRIPTION_ID=$1
+
+#echo ""
+#echo "  Background article: https://azure.microsoft.com/en-us/documentation/articles/resource-group-authenticate-service-principal"
+echo ""
+
+my_app_name_uuid=$(python -c 'import uuid; print str(uuid.uuid4())[:8]')
+MY_APP_NAME="app${my_app_name_uuid}"
+
+MY_APP_KEY=$(python -c 'import uuid; print uuid.uuid4().hex')
+
+my_app_id_URI="${MY_APP_NAME}_id"
+
+#check if the user has subscriptions. If not she's probably not logged in
+subscriptions_list=$(azure account list --json)
+subscriptions_list_count=$(echo $subscriptions_list | jq '. | length' 2>/dev/null)
+if [ $? -ne 0 ] || [ "$subscriptions_list_count" -eq "0" ]
+then
+  azure login
```

Comment:
This is not the right script, it's create_spn.sh at the root. This one needs to be removed.